src/helics/application_api/FederateInfo.cpp

```diff
@@ -211,13 +211,23 @@ static void loadFlags(FederateInfo& fi, const std::string& flags)
             fi.autobroker = true;
             continue;
         }
+        if (flg.empty()) {
+            continue; // LCOV_EXCL_LINE
+        }
         auto loc = validFlagOptions.find(flg);
         if (loc != validFlagOptions.end()) {
             fi.setFlagOption(propStringsTranslations.at(flg), true);
         } else {
+            if (flg.front() == '-') {
```

Comment:
How are flags starting with `--` handled?

Reply:
in this case they wouldn't match,  the `-` implies negation, so it would try to match `-flag` which would fail.

Reply:
Okay, I thought this was the `-` and `--` that go before command line args/flags.